components/extensions/src/main/kotlin/io/github/composegears/valkyrie/extensions/Any.kt

```diff
@@ -1,5 +1,16 @@
+@file:Suppress("NOTHING_TO_INLINE")
+
 package io.github.composegears.valkyrie.extensions
 
 inline fun <reified T : Any> Any?.safeAs(): T? = this as? T
 
 inline fun <reified T : Any> Any?.cast(): T = this as T
+
+inline infix fun <T> T?.or(default: T): T = this ?: default
+
+fun Any?.toStringOrNull(): String? {
+    return when (val value = toString()) {
+        "null" -> null
+        else -> value
+    }
+}
```

Comment:
```suggestion
```

I guess you want `this?.toString()`.

Reply:
Oh, yes

I tried `nullable.toString()` and it returns "null" 😄